src/plugins/divider/divider.yml

```diff
@@ -0,0 +1,28 @@
+divider:
+  name:
+    en: Divider
+    de: Verteiler
```

Comment:
Hm, that would be "distributor". Divider would be "Teiler", though it seems to me to not be very clear what the plugin does. Clearer would be "Trennline" (separator / separator line). Maybe "separator" would be also clearer in English, what do you think?

Reply:
I wasn't sure about divider or separator. Then I went with divider because AntD also calls it that 🤷‍♀️ 
We can change to separator, sure.

Reply:
Changed!

Reply:
:+1: